### ISSUE TITLE
NoWait for Az.Resources LRO

### ISSUE THREAD

**Issue by bgelens** [Azure/azure-powershell#9073]
## Description of the new feature

Like for Compute LRO cmdlets (#8846), add NoWait switches for Az.Resources LRO cmdlets

* [ ] New-AzResourceGroupDeployment
* [ ] New-AzDeployment

## Proposed implementation details (optional)

Return PSAzureOperationResponse with CorrelationId and OperationId

**Comment by markcowl** [comment]
In the fullness of time, this will be supported across long-running cmdlets for all services.  We should have a preview of this for a large number of services later this year.

**Comment by bgelens** [comment]
Awesome! You can close this issue if you like or keep it open if you want to track priorities based on customer feedback? For me these are the most important ones!

**Comment by slavizh** [comment]
No preview yet?

**Comment by markcowl** [comment]
Preview went out in October, with a refresh in January: https://www.powershellgallery.com/packages/Az/4.0.2-preview

**Comment by slavizh** [comment]
@markcowl Yes, saw that a little bit later after posted the reply. I guess we will wait for GA.

**Comment by shellby1** [comment]
it's nice feature
how is going on?
as 16+ version is released
